.devcontainer/conf/php/Dockerfile

```
@@ -0,0 +1,52 @@
+FROM php:8.2-fpm-alpine
+
+# Install tools
+RUN apk add --update --no-cache \
+    nodejs-current npm mysql-client curl patch gettext perl git \
+    zsh sudo vim htop && \
```

The `plugin-release` script may be used to build archives based on a specific commit, but this is really a rare usage of it. It may make the image of the container bigger for not much, an people could probably use a `docker run` command based on another image.
I do not know if we should provide this or not.

```suggestion
    zsh sudo vim htop && \
    # Install Python, pip and Python modules required by "plugin-release" script.
    apk add python3-dev py3-pip && \
    apk add libffi-dev libxml2-dev libxslt-dev && pip install lxml && \
    apk add git && pip install gitpython && \
    pip install PyGithub && \
    pip install termcolor && \
```